content/buy-navcoin/index.en.md

```diff
@@ -127,6 +133,36 @@ newTab="true"
           text="BTC + ETH / NAV"
           linkUrl="https://bitqist.com/exchange/nav-coin"
       >}}
+     {{< exchange
+          titleText="Vertbase"
+          imgSrc="/images/buy-navcoin/vertbase.jpg"
+          text="USD / NAV"
+          linkUrl="https://www.vertbase.com/"
+      >}}
+     {{< exchange
+          titleText="Altilly"
+          imgSrc="/images/buy-navcoin/altilly.jpg"
+          text="BTC / NAV"
+          linkUrl="https://www.altilly.com/asset/NAV"
+      >}}
+     {{< exchange
+          titleText="Nova Exchange"
+          imgSrc="/images/buy-navcoin/nova_exchange.jpg"
+          text="BTC + ETH / NAV"
+          linkUrl="https://bitqist.com/exchange/nav-coin"
```

Comment:
url should be Nova Exchange for all languages

Reply:
Good catch

Reply:
Fixed